### ISSUE TITLE
[a11y] Selection bar doesn't show in Windows High Contrast mode

### ISSUE THREAD

**Issue by taranir** [PolymerElements/paper-tabs#186]
### Description

When Windows 7 High Contrast mode is enabled, the selection bar is not visible. This is likely because the high contrast mode changes all background-color css styling to black (or the default background color for the mode), and all foreground color to white (or default foreground color).
### Expected outcome

The selected tab should have a visible selection indicator.
### Actual outcome

The selected tab looks the same as non-selected tabs.
### Live Demo

 http://i.imgur.com/9zVy5zo.png
### Steps to reproduce
1. Turn on Windows High Contrast mode, and open a Firefox 47 browser window.
2. Open a page with a paper-tab element.
3. Select a tab. 
### Browsers Affected

Firefox 47
